oauth2_http/javatests/com/google/auth/oauth2/UserCredentialsTest.java

```diff
@@ -504,6 +506,39 @@ public void fromStream_userNoRefreshToken_throws() throws IOException {
     testFromStreamException(userStream, "refresh_token");
   }
 
+  @Test
+  public void saveUserCredentials_saved_throws() throws IOException {
+    UserCredentials userCredentials = UserCredentials.newBuilder()
+            .setClientId(CLIENT_ID)
+            .setClientSecret(CLIENT_SECRET)
+            .setRefreshToken(REFRESH_TOKEN)
+            .build();
+    String filePath = System.getProperty("user.dir") + File.separator + "GOOGLE_APPLICATION_CREDENTIALS";
```

Comment:
As someone running these tests, I don't want this file to show up in my home directory when I run them. Put them under /tmp (use the Java function to auto-generated a file under /tmp)

Reply:
Updated to use temporary files.